includes/minify/class-html.php

```
@@ -361,6 +414,16 @@ private function safe_minify_js( string $attributes, string $content ): string {
 						// If the 'type' attribute doesn't exist, add a new one.
 						$attributes .= ' type="wppo/javascript" wppo-type="text/javascript"';
 					}
+
+					// Add strategy and priority data attributes for inline scripts.
+					$strategy = $this->get_delay_strategy_for_inline( $attributes, $content );
```

PHPStan (2.2.6)

[warning] Parameter `#1` $attributes of method PerformanceOptimise\Inc\Minify\HTML::get_delay_strategy_for_inline() expects string, string|null given.


(argument.type)